File: fast-react/src/common/enmus/teamresource-enum.js
export const teamresourceEnum = {
    DEMAND_GROUP: 1,
    RD_GROUP: 2,
    TEST_GROUP: 3,
    TECHNICAL_GROUP: 4,
    PROJECT_GROUP: 5,
    getName: (code) => {
        if (code === 1) return '需求组'
        if (code === 2) return '研发组'
        if (code === 3) return '测试组'
        if (code === 4) return '生产组'
        if (code === 5) return '实施组'
        return ''
    }
}

File: fast-boot/src/main/java/com/example/fastboot/common/enums/TeamResourceEnum.java
package com.example.fastboot.common.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

/**
 * @description:
 * @author: winl
 * @time: 2023/1/16 16:12
 */
@AllArgsConstructor
@Getter
public enum TeamResourceEnum {

    DEMAND_GROUP(1,"需求组"),
    RD_GROUP(2,"研发组"),
    TEST_GROUP(3,"测试组"),
    TECHNICAL_GROUP(4,"生产组"),
    PROJECT_GROUP(5,"实施组");
    private int code;
    private String name;

}
